proto/protocolapis/astria/protocol/transaction/v1/action.proto

```diff
@@ -254,9 +255,18 @@ message FeeChange {
     astria.protocol.fees.v1.TransferFeeComponents transfer = 13;
     astria.protocol.fees.v1.ValidatorUpdateFeeComponents validator_update = 14;
     astria.protocol.fees.v1.BridgeTransferFeeComponents bridge_transfer = 15;
+    astria.protocol.fees.v1.RecoverClientFeeComponents recover_client = 16;
   }
 }
 
 message IbcSudoChange {
   astria.primitive.v1.Address new_address = 1;
 }
+
+message RecoverClient {
+  // the client identifier for the client to be updated
+  string subject_client_id = 1;
+  // the substitute client identifier for the client which will replace the subject
+  // client
+  string substitute_client_id = 2;
```

Comment:
nit: I think subject & substitute are kinda confusing. Preference for `client_id` and `replacement_client_id`.

Reply:
don't really like just `client_id` as it's too ambiguous imo and will be more confusing reading the code. could do `client_id_to_replace` and `substitute_client_id`, or `subject_client_id` and `replacement_client_id`?